src/components/Insights/ScalingIssueInsight/mockData.ts

```diff
@@ -0,0 +1,185 @@
+import { InsightType, SpanInfo } from "../../../types";
+import { Duration } from "../../../globals";
+import {
+  SpanScalingBadlyInsight,
+  InsightCategory,
+  InsightScope,
+  CodeObjectDecorator,
+  InsightImportance,
+  RootCauseSpanInfo,
+  InsightSpecificity,
+  AffectedEndpoint
+} from "../types";
+
+export class MockedSpanScalingBadlyInsight implements SpanScalingBadlyInsight {
```

Comment:
it is better to have several sample objects instead of separate class, also u can use object deconstruction to reduce code amount

Reply:
done